synchronizer/synchronizer.go

```diff
@@ -109,16 +109,19 @@ func NewSynchronizer(
 		l1EventProcessors:       nil,
 		halter:                  syncCommon.NewCriticalErrorHalt(eventLog, 5*time.Second), //nolint:gomnd
 	}
-	L1SyncChecker := l2_sync_etrog.NewCheckSyncStatusToProcessBatch(res.zkEVMClient, res.state)
 
-	syncTrustedStateEtrog := l2_sync_etrog.NewSyncTrustedBatchExecutorForEtrog(res.zkEVMClient, res.state, res.state, res,
-		syncCommon.DefaultTimeProvider{}, L1SyncChecker)
+	if !isTrustedSequencer {
+		log.Info("Permissionless: creating L2 synchronization stuff")
```

Comment:
not sure if `stuff` word is a good one to put here in an `Info` level log message. Maybe we could be more explicit in what's being prepared, or something like:
"Permissionless: creating and Initializing L2 synchronization components"